includes/controllers/class.llms.controller.certificates.php

```diff
@@ -117,10 +117,34 @@ public function maybe_handle_reporting_actions() {
 			$this->download( $cert_id );
 		} elseif ( isset( $_POST['llms_delete_cert'] ) ) {
 			$this->delete( $cert_id );
+		} elseif ( isset( $_POST['llms_change_shareable_cert'] ) ) {
+			$this->change_shareable_settings( $cert_id, isset( $_POST['llms_is_shareable_cert'] ) );
 		}
 
 	}
 
+	/**
+	 * Change shareable settings of a certificate
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param int $cert_id WP Post ID of the llms_my_certificate
+	 * @param bool $is_allow Allow share the certificate or not
+	 * @return void
+	 */
+	private function change_shareable_settings( $cert_id, $is_allow ) {
+
+		$uid = get_current_user_id();
+		$cert = new LLMS_User_Certificate( $cert_id );
+
+		if ( $uid != $cert->get_user_id() && ! is_admin() ) {
+			return;
+		}
+
+		update_post_meta($cert_id, '_llms_allow_sharing', $is_allow ? 'yes' : 'no');
```

Comment:
Please use "allowed" in favor of "allow" so the variables read more naturally. 

```suggestion
	 * @param bool $is_allowed Allow share the certificate or not
	 * @return void
	 */
	private function change_shareable_settings( $cert_id, $is_allowed ) {

		$uid = get_current_user_id();
		$cert = new LLMS_User_Certificate( $cert_id );

		if ( $uid != $cert->get_user_id() && ! is_admin() ) {
			return;
		}

		update_post_meta($cert_id, '_llms_allow_sharing', $is_allowed ? 'yes' : 'no');
```